modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -3607,64 +3571,143 @@ enum GmosNorthGrating {
   R150_G5308
 }
 
-# GMOS North Long Slit mode
-type GmosNorthLongSlit {
-  # GMOS North Long Slit basic configuration
-  basic: GmosNorthLongSlitBasic!
-
-  # GMOS North Long Slit advanced configuration
-  advanced: GmosNorthLongSlitAdvanced
-}
+# GMOS Long Slit Observing Mode common fields
+interface GmosLongSlitObservingMode {
```

Comment:
An interface with the common GMOS long slit observing mode values. For any property, such as `xBin`, there are three fields:

1. the property value that will be used (e.g., `xBin`)
2. the default value of that property if not explicitly set (e.g., `defaultXBin`)
3. the (optional) explicit value chosen for this property by the user (e.g., `explicitXBin`)

Only the `explicit___` value appears in the input.  The others are calculated or set from constants.

Reply:
If you only care about getting the value that will be used, the main property value should be consulted.  For x-binning it would be just `xBin`.  If you want to know what the default value we use will be (which for x-binning depends on the grating, target and image quality) use `defaultXBin`.  A UI could compare the `xBin` to `defaultXBin` to determine whether the property has been changed from its nominal value.  It could offer a way to abandon customizations and go back to the defaults.